### DISCUSSION TITLE
How to force a window resize when another client with a different resolution is connected to the session?

### DISCUSSION THREAD

**Discussion by MathieuMoalic** [zellij-org/zellij#3124] [General]
I have a host machine with a 1920x1200 monitor on which I have a zellij session running. I use `mosh` on my laptop (1920x1080p) to connect to the host and attach to the session. Because mosh keep the connection "alive", the session on the host is kept at the laptop's resolution. Resizing the windows, re-attaching the session will not match the session's window size to the terminal window size (i.e. there is a blank space at the botttom). 
Is there a way to force a resize to the current terminal size ?

**Comment by giyany** [comment]
No, but it should be possible to disconnect clients starting from the next version.
